### PR TITLE
feat (send-notification): update post scan api to accept reply url

### PR DESCRIPTION
**Pull request by haonliu**
Accept reply url sending with the request and save it into the database. 

Will check if the reply url is valid if the user does provide it. If not, will return an error response.

Api version management refactor will be a separate PR. We might need to put that in the service config.

### CODE REVIEW

File: packages/web-api/src/controllers/scan-request-controller.ts [hunk 1]
```diff
@@ -88,6 +85,7 @@ export class ScanRequestController extends ApiController {
         this.logger.trackEvent('BatchScanRequestSubmitted', null, measurements);
     }
 
+    // tslint:disable-next-line: no-any
```
Comment (devaradhanm) [inline packages/web-api/src/controllers/scan-request-controller.ts]: avoid no-any in product code. add strict typings

File: packages/web-api/src/controllers/scan-request-controller.spec.ts [hunk 2]
```diff
@@ -74,13 +74,22 @@ describe(ScanRequestController, () => {
     }
 
     describe(ScanRequestController, () => {
-        it('rejects request with large payload', async () => {
-            context.req.rawBody = JSON.stringify([{ url: '' }, { url: '' }, { url: '' }, { url: '' }]);
+        // it('rejects request with large payload', async () => {
```
Comment (devaradhanm) [inline packages/web-api/src/controllers/scan-request-controller.spec.ts]: remove commented code

File: packages/service-library/src/web-api/api-contracts/scan-run-request.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 export interface ScanRunRequest {
     url: string;
-
+    replyUrl?: string;
```
Comment (manishsat) [inline packages/service-library/src/web-api/api-contracts/scan-run-request.ts]: I think WCP team sends {url, priority} and they will send  {url, priority, replyUrl}, plz verify that the sequence and I am happy to know whether sequence matters or not.

File: packages/web-api/src/controllers/scan-request-controller.spec.ts (hunk 2 above)
Comment (manishsat) [reply packages/web-api/src/controllers/scan-request-controller.spec.ts]: +1

**Comment by devaradhanm** [comment]
update pr title to feat (send-notification): ....

File: packages/service-library/src/web-api/api-contracts/scan-run-request.ts (hunk 3 above)
Comment (haonliu) [reply packages/service-library/src/web-api/api-contracts/scan-run-request.ts]: I don't think the order matters, cause we are parsing them as JSON object.

**Pull request merged**